src/cmd/linuxkit/pkgsrc/docker.go

```diff
@@ -0,0 +1,90 @@
+package pkgsrc
+
+// Thin wrappers around Docker CLI invocations
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+const debugDockerCommands = false
+
+const dctEnableEnv = "DOCKER_CONTENT_TRUST=1"
+
+type dockerRunner struct {
+	dct   bool
+	cache bool
+}
+
+func newDockerRunner(dct, cache bool) dockerRunner {
+	return dockerRunner{dct: dct, cache: cache}
+}
+
+func (dr dockerRunner) command(args ...string) error {
+	cmd := exec.Command("docker", args...)
```

Comment:
I would find `docker` with lookpath and error up front of the docker binary is not found in a more helpful way.

Reply:
I broke the name and the result is:
```
$ linuxkit pkg build pkg/init
exec: "dockerx": executable file not found in $PATH
```

Reply:
yes but it doesnt give you a helpful message like "linuxkit pkg requires docker to be installed"

Reply:
Is that not readily apparent from "docker not found in $PATH" ?

Reply:
No. Because I didnt try to run docker and I dont know the implementation currently shells out, this is unexpected.

Reply:
I don't understand the argument. To me `executable file not found in $PATH` is sufficient

Reply:
I mostly agree with @rn but I think it would be sufficient to check for `exec.ErrNotFound` rather than doing a manual `LookPath` so the impact might not be too bad.